convert2rhel/actions/system_checks/package_updates.py

```diff
@@ -54,12 +67,17 @@ def run(self):
             # repositories, we use this to catch exceptions when verifying if there is any packages to update on the system.
             # Beware that the `RepoError` exception is based on the `pkgmanager` module and the message sent to the output
             # can differ depending if the code is running in RHEL7 (yum) or RHEL8 (dnf).
-            logger.warning(
+            warning_message = (
                 "There was an error while checking whether the installed packages are up-to-date. Having an updated system is"
                 " an important prerequisite for a successful conversion. Consider verifyng the system is up to date manually"
                 " before proceeding with the conversion."
             )
-            logger.warning(str(e))
+            logger.warning(warning_message + " %s" % str(e))
```

Comment:
Note: Better to way to do this is to assemble this in the warning_message rather than both here and in the add_message() call.  (No need to do this now, though.)

Reply:
Adding this